python/cuspatial/cuspatial/core/spatial/bounding.py

```diff
@@ -51,23 +51,21 @@ def polygon_bounding_boxes(polygons: GeoSeries):
     # by combining the geometry offset and parts offset of the multipolygon
     # array.
 
-    poly_offsets = polygons.polygons.part_offset.take(
-        polygons.polygons.geometry_offset
-    )
+    poly_offsets = polygons.polygons.part_offset
```

Comment:
I'm not sure if I understand why you need to revert this change. This should enable support for both polygon and multipolygon support regardless. Does the previous code break any existing code?